docs/basics/markdown.md

```diff
@@ -26,6 +26,20 @@ span: 2
 
 Catalog provides many specimen types, from images, hints, code snippets to fully interactive HTML and React component specimens. For a full list of all specimens and their features, go to the [Specimen documentation](/specimens)
 
+## Links
+
+Use regular Markdown link to pages within your Catalog documentation.
+
+Each page section can be linked to directly by adding a hash to your link. Click on the link icon next to a heading to jump to that section.
+
+```code|lang-markdown
+- [A link to a page](/specimens)
+- [A link to a page section](/specimens#content)
+- [A link to section on the same page](#example-page)
```

Comment:
This is nice, but I wonder whether it should be less automagical. It makes me slightly nervous that many pages will link to another page and someone editing that other page won't know that they might break a lot of links when they change a title. I'm pretty sure that these things will happen in the worst moments …

One idea is to use something along the Markdown footnote syntax: `[Aaa Bbb Ccc][abc]`.